requirements.txt

```diff
@@ -1,4 +1,5 @@
 -e git+https://github.com/LUMC/magic-python.git#egg=Magic_file_extensions
+-e git+https://git.lumc.nl/mutalyzer/extractor.git@3148fad6b081c36e9675494f4896ba60f14707a9#egg=extractor
```

Comment:
I specified the current master commit hash here, to prevent accidental breakeage of our servers.

This is fine for now, but ideally Jonathan would tag the extractor releases.